src/integrations/checkout.js

```diff
@@ -31,7 +31,8 @@ export function determineUrl(env : string, fundingSource : ?string, payment : st
         return config.guestUrls[env];
     }
 
-    if (fundingSource === FUNDING.IDEAL) {
```

Comment:
This'll work for now. In future we should probably define a map of funding method -> url though.

Reply:
I agree. Was thinking about this.